src/PHPStan/Reflection/CallReflectionResolver.php

```diff
@@ -13,17 +14,25 @@
 use PHPStan\Broker\FunctionNotFoundException;
 use PHPStan\Reflection\FunctionReflection;
 use PHPStan\Reflection\MethodReflection;
+use PHPStan\Reflection\Native\NativeFunctionReflection;
 use PHPStan\Reflection\ParametersAcceptor;
 use PHPStan\Reflection\ParametersAcceptorSelector;
 use PHPStan\Reflection\ReflectionProvider;
-use PHPStan\Type\ClosureType;
+use PHPStan\TrinaryLogic;
+use PHPStan\Type\Constant\ConstantStringType;
 use Rector\NodeTypeResolver\Node\AttributeKey;
 use Rector\NodeTypeResolver\NodeTypeResolver;
 use Rector\PhpParser\Node\Resolver\NameResolver;
-use Rector\PHPStan\Reflection\Php\ClosureInvokeMethodReflection;
 
 final class CallReflectionResolver
 {
+    /**
+     * Took from https://github.com/phpstan/phpstan-src/blob/8376548f76e2c845ae047e3010e873015b796818/src/Type/Constant/ConstantStringType.php#L158
+     *
+     * @see https://regex101.com/r/IE6lcM/1
```

Comment:
Great! The content it matches doesn't matches should be also filled,
e.g. https://regex101.com/r/IE6lcM/3

Reply:
Done.